public_suffix_list.dat

```diff
@@ -11231,6 +11231,11 @@ cnpy.gdn
 // Submitted by Moritz Marquardt <git@momar.de>
 codeberg.page
 
+// CodeSandbox B.V. : https://codesandbox.io
+// Submitted by Ives van Hoorne <ives@codesandbox.io>
```

Comment:
Please use a non-personal e-mail address.

Reply:
Would a company wide email like abuse@codesandbox.io be better?

Reply:
I've updated the email and sorted domains!